### PR TITLE
AA-340: Update the RIP-7560 with the EIP-7702 transaction type fields

### PR DESCRIPTION
**Pull request by forshtat**

### CODE REVIEW

File: RIPS/rip-7560.md [hunk 1]
```diff
@@ -217,10 +225,20 @@ struct TransactionType4 {
     bytes paymasterData;
     bytes callData;
     bytes signature;
+    address[] authorizationList;
+    bool[] authorizationListStatus;
```
Comment (drortirosh) [inline RIPS/rip-7560.md:281]: maybe better to provide `authorizedList` and `failedAuthorizationList` (which is expected to be empty)

File: RIPS/rip-7560.md [hunk 2]
```diff
@@ -61,7 +61,7 @@ smart contracts. This, however, is impossible without an addition of Native Acco
 | Name                  | Value                                                                           |
 |-----------------------|---------------------------------------------------------------------------------|
 | FORK_BLOCK            | 	TBD                                                                            |
-| AA_TX_TYPE            | 4                                                                               |
+| AA_TX_TYPE            | 126                                                                             |
```
Comment (sm-stack) [inline RIPS/rip-7560.md]: I'm afraid that 126 is a deposit tx type at Optimism. Relative docs at optimism specs [here](https://specs.optimism.io/protocol/deposits.html#the-deposited-transaction-type).

File: RIPS/rip-7560.md (hunk 1 above)
Comment (forshtat) [reply RIPS/rip-7560.md:281]: Just talked to @shahafn about that - we could not see how would it make it better? Appears to provide the exact same information (except, maybe, losing the ordering - which might even be a downside in some edge-cases) and not improving anything.
I prefer the `bool[]` approach.

**Review by shahafn — APPROVED** [review]
authorizationList

**Pull request merged**